CHANGELOG.md

```diff
@@ -25,7 +25,9 @@
   [#368](https://github.com/vmware/go-vcloud-director/pull/368)
 * Added methods Org.QueryVmList and Org.QueryVmById to find VM by ID in an Org
   [#368](https://github.com/vmware/go-vcloud-director/pull/368)
-  
+* Added methods `NsxtEdgeGateway.UpdateNsxtFirewall()`, `NsxtEdgeGateway.GetNsxtFirewall()`, `nsxtFirewall.DeleteAll()`,
+  `nsxtFirewall.DeleteById` [#381](https://github.com/vmware/go-vcloud-director/pull/381)
+
 BREAKING CHANGES:
 * Added parameter `description` to method `vdc.ComposeRawVapp` [#372](https://github.com/vmware/go-vcloud-director/pull/372)
```

Comment:
Though unlrelated to this PR, I just noticed this line. Do we break the semver aspect of the govcd versioning here?

CC: @dataclouder

Reply:
Yep - this happened as part of #372